test/integration/registration_display_modes_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class GameboardDisplayModesTest < ActionDispatch::IntegrationTest
+  include TeamsHelper
+  include Devise::Test::IntegrationHelpers
+
+  test "registration blocked when registration closed" do
+    create(:active_game, board_layout: :jeopardy, registration_enabled: false)
+    
+    get "/users/new"
+    assert_response :redirect
```

Comment:
This should be done as part of `test/controller/registrations_controller_test.rb`, the only thing that it would be reasonable to verify here is whether or not "Register" button is properly removed from the site itself. Integration tests actually stand up a full webserver to run the tests, controller tests do not.